src/ore/Cargo.toml

```diff
@@ -23,15 +23,15 @@ ctor = { version = "0.1.23", optional = true }
 either = "1.8.0"
 futures = { version = "0.3.21", optional = true }
 once_cell = "1.13.0"
-# This isn't directly imported by anything, but it's required at link time. The
-# vendored feature is transitively depended upon by tokio-openssl.
```

Comment:
I would adjust this to mention that its required at link time AND as an import

Reply:
Thoughts on the new wording?

Reply:
Something like

```
# Even if this is directly imported (it currently is), its unconditionally required at link time,
# as the vendored feature is transitively depended on by `tokio-openssl`